scripts/render.py

```diff
@@ -72,7 +72,7 @@ def _render_trajectory_video(
         output_image_dir.mkdir(parents=True, exist_ok=True)
     else:
         install_checks.check_ffmpeg_installed()
-    semantics_model2output = pipeline.model.semantics_model2output.copy().cpu()
+    semantics_model2output = pipeline.model.semantics_model2output.detach().cpu()
```

Comment:
nice, did this make it faster?

Reply:
I remember the `.copy()` raises error, therefore I change to `.detach()`